site/content/blog/dagger-and-kapp.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Kapp and Dagger"
+slug: dagger-and-cloak
+date: 2022-07-30
+author: Renu Yarday
+excerpt: "In this article, we will explore how to leverage kapp in a Dagger pipeline."
+image: /img/kapp.svg
+tags: ['kapp', 'dagger']
+---
+
+# Running kapp in a Dagger pipeline
+
+In this article we will explore how to leverage kapp in a Dagger pipeline.
+
+### What is Dagger?
+Dagger is a portable devkit to build powerful CI/CD pipelines quickly and run them anywhere. Read more about Dagger [here](https://dagger.io/).
+
+### Introducing kapp package for Dagger
+Do you want to observe your Kubernetes deployments are going smoothly in your Dagger pipeline? Then we highly recommend trying out [kapp deploy](https://github.com/dagger/dagger/tree/main/pkg/universe.dagger.io/alpha/kubernetes/kapp). Kapp is now available as an alpha package with Dagger and can be easily consumed in your CI/CD. 
```

Comment:
> Do you want to observe your Kubernetes deployments are going smoothly in your Dagger pipeline?

this to me implies that kapp isnt used to actually do the deploy. proposing to change to...

`Do you want to deploy your Kubernetes configuration from your Dagger pipeline?`

Reply:
changing to  - Do you want to deploy your Kubernetes configuration from your Dagger pipeline? Along with applying changes safely and predictably, watching resources as they converge.